Add yosensi folder with all devices

yosensi opened the pull request:
<!--
Thanks for submitting a pull request. Please fill the template below,
otherwise we will not be able to process this pull request.
-->

#### Summary
<!--
A short summary, referencing related issues:
Closes #0000, References #0000, etc.
-->

Add folder with all devices with yamal files

#### Changes
<!-- What are the changes made in this pull request? -->

- ...
- ...

#### Notes for Reviewers
<!--
NOTE: This section is optional.

Motivate briefly why it is implemented this way, if that deviates from the
implementation proposal in the referenced issues.
- How should your reviewers approach this pull request?
- @mention reviewers with special requests or questions for them
-->

...

#### Release Notes
<!--
NOTE: This section is optional.

Any notes that we need to include in the Release Notes for the next release.
These notes are formatted as bullet points, written in past tense, and will be
combined with the labels of this Pull Request.

Always mention changes in API.
-->

- ...

Jaime-Trinidad commented on vendor/yosensi/yo-360-profile-au.yaml:
```diff
@@ -0,0 +1,52 @@
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
@yosensi if MAC version is 1.0.3 `Regional Parameters` might be: `RP001-1.0.3-RevA` check which one is the correct one and change it

Jaime-Trinidad commented on vendor/yosensi/yo-360-profile-us.yaml:
```diff
@@ -0,0 +1,52 @@
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
same case `MAC` and `Regional Parameters`

Jaime-Trinidad commented on vendor/yosensi/yo-agribox-profile-au.yaml:
```diff
@@ -0,0 +1,52 @@
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
review it

Jaime-Trinidad commented on vendor/yosensi/yo-agribox-profile.yaml:
```diff
@@ -0,0 +1,52 @@
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
review

Jaime-Trinidad commented on vendor/yosensi/yo-airflow-pro-profile-us.yaml:
```diff
@@ -0,0 +1,52 @@
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
check

Jaime-Trinidad commented on vendor/yosensi/yo-airflow-pro-profile.yaml:
```diff
@@ -0,0 +1,52 @@
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
check

Jaime-Trinidad commented on vendor/yosensi/yo-analog-profile-au.yaml:
```diff
@@ -0,0 +1,52 @@
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
Review

Jaime-Trinidad commented on vendor/yosensi/yo-analog-profile-us.yaml:
```diff
@@ -0,0 +1,52 @@
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
Review

Jaime-Trinidad commented on vendor/yosensi/yo-analog-profile.yaml:
```diff
@@ -0,0 +1,52 @@
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
Review

Jaime-Trinidad commented on vendor/yosensi/yo-distance-profile-au.yaml:
```diff
@@ -0,0 +1,52 @@
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
Review

Jaime-Trinidad commented on vendor/yosensi/yo-distance-profile-us.yaml:
```diff
@@ -0,0 +1,52 @@
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
Review

Jaime-Trinidad commented on vendor/yosensi/yo-h2o-profile-au.yaml:
```diff
@@ -0,0 +1,52 @@
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
Review

Jaime-Trinidad commented on vendor/yosensi/yo-h2o-profile-us.yaml:
```diff
@@ -0,0 +1,52 @@
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
```
Review

Jaime-Trinidad commented on vendor/yosensi/yo-h2o.yaml:
```diff
@@ -0,0 +1,86 @@
+    version: '3.1.0'
+    numeric: 1
+    hardwareVersions:
+      - '1.3'
```
check the hardware version

Jaime-Trinidad commented on vendor/yosensi/yo-pure-pro.yaml:
```diff
@@ -0,0 +1,93 @@
+<<<<<<< HEAD
+=======
+    # Corresponding hardware versions (optional)
+>>>>>>> b13e0bc5a0e9016e8a419fc125a3f6c2f96cae58
```
Review

Jaime-Trinidad commented:
@yosensi please review the comments and check all your profile files, between the `MAC version` and `Regional Parameters`

yosensi commented:
@Jaime-Trinidad I made review and changed MAC version for all devices. I added by accident .cpp file to one folder. I already did second commit that delete those files.

yosensi commented:
Hello is there something more I need to do to upload Yosensi folder?
